### PR TITLE
Update phone images

### PR DESCRIPTION
**Pull request by nshonda**
Related Issue:
https://airtable.com/appsEN1RgrUk3oyon/tbluJ0kJjMnu1rtsx/viwEjc1BEgM0pj72s/rec5bgyokbgYCcZJK?blocks=hide

### CODE REVIEW

File: src/components/partials/SliceZone/components/HowItWorksSlice/index.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ const HowItWorksSlice = ({ slice }) => {
               height={image.dimensions.height}
               alt={image.alt}
               title={image.title}
+              layout="fixed"
```
Comment (quaardvark) [inline src/components/partials/SliceZone/components/HowItWorksSlice/index.js:37]: This will break a lot of things. Is it necessary?

Comment (quaardvark) [reply src/components/partials/SliceZone/components/HowItWorksSlice/index.js:37]: Just visiting the home page makes it really obvious - https://marketing-git-update-phone-images-rent-avail.vercel.app/

Comment (nshonda) [reply src/components/partials/SliceZone/components/HowItWorksSlice/index.js]: sorry, missed this

File: pages/tenants/digital-leasing/data/second-how-it-works.js [hunk 2]
```diff
@@ -20,8 +20,8 @@ export default {
       embed: {},
       image: {
         dimensions: {
-          height: 437,
-          width: 305,
+          height: 360,
+          width: 184,
```
Comment (quaardvark) [inline pages/tenants/digital-leasing/data/second-how-it-works.js:24]: These sizes should match the intrinsic image size

**Review by quaardvark — CHANGES_REQUESTED** [review]

**Pull request merged**